src/templates/common/mediaImage/index.tsx

```diff
@@ -67,6 +67,7 @@ export const MediaImage = (
       title={props.title}
       className={props.className}
       style={props.style}
+      loading={props.loading}
```

Comment:
This allows us to pass the `loading` prop to Next.js's `<Image />` component. We do this to avoid lazy loading in the image below.